components/log-viewer-webui/client/src/components/Layout/MainLayout.tsx

```diff
@@ -40,7 +40,7 @@ const MainLayout = () => {
                 collapsed={collapsed}
                 collapsible={true}
                 theme={"light"}
-                width={"150"}
+                width={200}
```

Comment:
```suggestion
                width={150}
```

I actually think 200 is too big compared to 150. Maybe it'll look better once we add more sidebar elements.